src-tauri/tauri.conf.json

```diff
@@ -61,7 +61,10 @@
         "width": 1000,
         "height": 700,
         "resizable": true,
-        "fullscreen": false
+        "fullscreen": false,
+        "center": true,
+        "focus": true,
+        "visible": true
       }
     ],
```

Comment:
It didn't open a splashscreen when I tried these changes locally and I think this is why: wouldn't the splashscreen not need to be added here?

Reply:
The initial flow was you would see the splash screen while we start the executor, once we have it running we would close it and then open the main window, but because of this bug https://github.com/tauri-apps/tauri/issues/4149 hiding and showing window would not work on linux so I removed it until it gets fixed.

Reply:
Tauri team get it fixed just now.